projects/ngrx.io/content/guide/router-store/configuration.md

```diff
@@ -41,14 +41,14 @@ import { Params, RouterStateSnapshot } from '@angular/router';
 import { RouterStateSerializer } from '@ngrx/router-store';
 
 export interface RouterStateUrl {
-  url: string;
-  params: Params;
-  queryParams: Params;
+url: string;
```

Comment:
Seems like something weird has happened here with the formation, could you revert this change please?